backend/dataall/tasks/data_sharing/share_managers/lf_share_manager.py

```diff
@@ -60,11 +60,14 @@ def get_share_principals(self) -> [str]:
         """
         principals = [f"arn:aws:iam::{self.target_environment.AwsAccountId}:role/{self.share.principalIAMRoleName}"]
         if self.target_environment.dashboardsEnabled:
-            q_group = Quicksight.get_quicksight_group_arn(
-                self.target_environment.AwsAccountId
+            group = Quicksight.create_quicksight_group(
+                self.target_environment.AwsAccountId, Quicksight.DEFAULT_GROUP_NAME
             )
-            if q_group:
-                principals.append(q_group)
+            if group and group.get('Group', {}).get('Arn'):
+                q_group = group.get('Group', {}).get('Arn')
```

Comment:
group and q_group make the code less readable, can we name it group_arn in here?